app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less

```diff
@@ -998,6 +998,15 @@
                 }
             }
         }
+
+        .block-compare {
+            .action {
+                &.delete {
+                    left: 0;
+                    right: auto;
+                }
+            }
+        }
     }
```

Comment:
Also please add a blank line after a selector compare.wrapper around line 1010
for code styles